websites/A/AnimeDrive/metadata.json

```
@@ -6,20 +6,23 @@
 	},
 	"service": "AnimeDrive",
 	"description": {
-		"en": "AnimeDrive is an anime streaming website with Hungarian subs/dubs"
+		"en": "AnimeDrive is an anime streaming website with Hungarian subs/dubs",
+		"hu": "Az AnimeDrive egy anime streaming weboldal Magyar felirattal/szinkronnal"
 	},
 	"url": [
 		"animedrive.hu",
 		"www.animedrive.hu"
 	],
-	"version": "1.0.1",
-	"logo": "https://cdn.rcd.gg/PreMiD/websites/A/AnimeDrive/assets/logo.png",
-	"thumbnail": "https://cdn.rcd.gg/PreMiD/websites/A/AnimeDrive/assets/thumbnail.png",
+	"version": "1.0.2",
+	"logo": "https://cdn.discordapp.com/icons/702098578251841577/be50b5dbcb30bd99512b6df29284db0c.webp",
```

Logos should be 512x512px. You may need to upload the large image to another image source to avoid issues with query parameters with the current schema.
```suggestion
	"logo": "https://i.imgur.com/VlBtvJo.png",
```

---

fixed